docs/dg/dev/acp/integrate-acp-payment-apps-with-spryker-oms-configuration.md

```diff
@@ -31,55 +37,126 @@ The main OMS file includes `<subprocesses>` – similar to libraries or building
 </subprocesses>
 ```
 
-Each process in the `<subprocesses>` section has a start state, one or more final states, and the states are linked to each other in the main State Machine .xml file.
+Each process in the `<subprocesses>` section has a start state, one or more final states, and the states are linked to each other in the main state machine XML file.
 
-![default-oms](https://spryker.s3.eu-central-1.amazonaws.com/docs/dg/dev/acp/integrate-acp-payment-apps-with-spryker-oms-configuration/default-oms.png)
+Other default subprocesses:
```

Comment:
Why are they `other`? We don't mention any default subprocesses before that

Reply:
Changed.